lib/country_codes.rb

```diff
@@ -292,6 +292,13 @@
     NO
   )
 
+LATAM_COUNTRY_CODES = %w(
```

Comment:
Is this actually all LATAM country codes?  Or just a subset for a specific purpose?

We seem to have a longer list of LATAM country codes at https://github.com/code-dot-org/code-dot-org/blob/staging/lib/country_regions.rb#L4

Reply:
This is a subset of LATAM countries as defined in [pegasus/sites.v3/hourofcode.com/i18n/countries.json](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/hourofcode.com/i18n/countries.json). That file and the [`hoc_canonicalized_i18n_path `](https://github.com/code-dot-org/code-dot-org/blob/8a6f853f7d2abe3664520d6950c13c83740486b6/pegasus/helpers/hourofcode_helpers.rb#L48) function control what version of the HoC page users from different countries will see.

Reply:
It shouldn't hurt to extend the list to include all LATAM countries.

Reply:
If it's a subset, should we use a variable name that makes that clear?

Reply:
I've updated the list to include all LATAM countries listed in `country_regions.rb`.